server/public/styles.css

```diff
@@ -0,0 +1,71 @@
+.page {
+  height: 100vh;
+  width: 100%;
+  background: rgb(147, 32, 224);
+  background: linear-gradient(0deg, rgba(147, 32, 224, 1) 0%, rgba(12, 0, 83, 1) 100%);
+  display: flex;
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.form-container {
+  background: rgb(12, 0, 83);
+  border-radius: 10px;
+  width: 345px;
+  padding: 20px 50px;
+}
+
+.form-title {
+  margin: 0 auto;
+  margin-bottom: 0.7rem;
+  font-weight: 400;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.center-content {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.text-box {
+  padding: 5px;
+  border-radius: 4px;
+  margin-bottom: 0.7rem;
+  color: rgb(12, 0, 83);
+}
+
+.text-box:focus {
+  outline-color: #9320e0;
+}
+
+.form-btn {
+  border: 0;
+  border-radius: 5px;
+  color: white;
+  font-size: 1rem;
+  padding: 3px 5px;
+  min-width: 105px;
```

Comment:
please remove this `min-width`. let me know if there's a specific problem you're trying to solve.